packages/react/src/ReactCurrentOwner.js

```diff
@@ -21,6 +22,7 @@ const ReactCurrentOwner = {
    * @type {ReactComponent}
    */
   current: (null: null | Fiber),
+  currentDispatcher: (null: null | Dispatcher),
```

Comment:
This is a very sensitive protocol since version mismatches are not uncommon. So we should think a bit more about what we want this protocol to be before the next release.

Reply:
Right. Do we warn about version mismatches? If not, seems like we should.

Reply:
We currently don’t. Last time we discussed this, we decided to try supporting React 16.0.0 in any 16.x renderer.

Is this going to break?